xmtp-labs/xmtp-inbox-web
# Add "Create Issue" prompt from app that directs to GitHub Issue. #182

## Description
The issue reported in #182 requested the addition of a "Create Issue" prompt from the app that directs to GitHub Issues.

## Solution
I have added a "Report Issue" button to the sidebar, which will redirect users to GitHub Issues. This will enable users to report issues easily and quickly, without having to navigate to GitHub themselves.

## How to Test
1. Open the app and navigate to the sidebar.
2. Look for the "Report Issue" button and click it.
3. Verify that the user is redirected to GitHub Issues.
4. Test that the prompt works correctly and allows users to report issues.

## Closing
With the addition of the "Report Issue" button, users can now easily report issues from the app. This should help to streamline the process of issue reporting and ultimately lead to a better user experience.
